pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -166,6 +177,36 @@ func (s *Service) ReconcileLoadBalancer(clusterName string, openStackCluster *in
 	return nil
 }
 
+func (s *Service) assignNeutronLbaasAPISecGroup(clusterName string, lb *loadbalancers.LoadBalancer) error {
+	neutronLbaasSecGroupName := fmt.Sprintf("%s-cluster-%s-secgroup-%s", networking.SecGroupPrefix, clusterName, networking.NeutronLbaasSuffix)
```

Comment:
We can do not exporting SecGroupPrefix and NeutronLbaasSuffix.


This line:
neutronLbaasSecGroupName := networking.GetNeutronLBaasSecGroupName(clusterName)


Add a following function in pkg/cloud/services/networking/securitygroups.go


```go

func GetNeutronLBaasSecGroupName(clusterName string) string {
 return fmt.Sprintf("%s-cluster-%s-secgroup-%s", secGroupPrefix, clusterName, neutronLbaasSuffix)
}